dashboard/src/index.css

```
@@ -1009,6 +1078,28 @@
+[data-plugin-list-switch='true'] {
+  width: 1.25rem !important;
+  height: 2.25rem !important;
+  flex-direction: column !important;
+  align-items: center !important;
+  justify-content: flex-start !important;
+}
+
+[data-plugin-list-switch='true'][data-state='checked'] [data-dashboard-switch-thumb='true'] {
+  --tw-translate-x: 0 !important;
+  --tw-translate-y: 1rem !important;
+  translate: 0 1rem !important;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)